Docs/reference/data/mongodb.toml

```diff
@@ -1,5 +1,5 @@
 githubRepo = "mongo-csharp-driver"
 githubBranch = "master"
-currentVersion = "2.11"
+currentVersion = "2.12"
```

Comment:
Shouldn't this remain at 2.11 for now?

Reply:
Probably. Initially I thought it was to build the docs for the new 2.12 branch, but it is probably used to build the landing page. I'll change it and deploy the built docs to my `gh-pages` so we can verify.